src/js/index.js

```diff
@@ -0,0 +1,96 @@
+import { $ } from "./utils/dom.js";
+import ApiMethod from "./api/index.js";
+import menuCountTemplate from "./template/menuCountTemplate.js";
+import categoryNameTemplate from "./template/categoryNameTemplate.js";
+import menuItemsTemplate from "./template/menuItemsTemplate.js";
+
+let categoryName = "espresso";
+const menuNameInput = $("#espresso-menu-name");
+const menuList = $("#espresso-menu-list");
+
+const addMenuName = async () => {
+  if (menuNameInput.value === "") {
+    return;
+  }
```

Comment:
```suggestion
  if (!menuNameInput.value) {
    return;
  }
```

위와 같이 boolean으로 표현하면 다른 예외 사항도 함께 처리할 수 있을 것 같아요 😄

Reply:
넵, 감사합니다. 한 층, 깔끔해졌어요.